test/unit/core/dom/layout/test-viewport-observer.js

```diff
@@ -116,18 +116,6 @@ describes.sandboxed('DOM - layout - Viewport Observer', {}, (env) => {
       expect(el1Events).eql([false]);
     });
 
-    it('Observing twice with the same callback is fine, but unique ones throw', () => {
```

Comment:
Last piece of feedback before I think this is good to go: we need new unit tests for the new functionality of handling multiple callbacks per node

Reply:
Done.